### ISSUE TITLE
Turn favicon red if there are any failed builds.

### ISSUE THREAD

**Issue by olymk2** [drone/drone-ui#290]
Another small Idea/Improvement, I really like it when the favicon changes colour this is a nice little touch it would be nice if when the build has finished it turned red if there was a failed build and went black or green when everything is good.

This way from a glance you can know if something needs looking at or not.

**Comment by RayBB** [comment]
Agreed, additionally it would be nice if the favicon was based on the state of the current build page you're on rather than just all recent builds.

**Comment by RayBB** [comment]
I wrote a small script to fix the issue. I anyone can install it in their browser.

https://github.com/raybb/drone-ui-alerts
